packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimePickerContent.test.tsx

```diff
@@ -1,11 +1,12 @@
+import { render, screen } from '@testing-library/react';
 import React from 'react';
 import { shallow } from 'enzyme';
```

Comment:
Could we also refactor other tests to use RTL? Seems weird to use two different libs in the same file.